packages/docusaurus-plugin-content-docs/src/props.ts

```diff
@@ -137,7 +137,7 @@ export function toTagDocListProp({
 }: {
   allTagsPath: string;
   tag: VersionTag;
-  docs: Pick<DocMetadata, 'id' | 'title' | 'description' | 'permalink'>[];
```

Comment:
as far as I remember the goal here was to avoid adding all docs attributes to tag pages as only a few attributes are displayed there => reduce tag page load time

in practice it seems doc props were not filtered, so each tag list page receives much more data than it really needs

Reply:
They are always filtered? See the code below. It's just that the function's argument is actually the full doc metadata